modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts

```diff
@@ -98,20 +98,20 @@ export async function parseI3STileContent(
     }
   }
 
-  content.material = makePbrMaterial(tile.materialDefinition, content.texture);
+  content.material = makePbrMaterial(tileOptions.materialDefinition, content.texture);
   if (content.material) {
     content.texture = null;
   }
 
-  return await parseI3SNodeGeometry(arrayBuffer, tile, content, tileset, options);
+  return await parseI3SNodeGeometry(arrayBuffer, tileOptions, content, tilesetOptions, options);
 }
 
 /* eslint-disable max-statements */
 async function parseI3SNodeGeometry(
   arrayBuffer: ArrayBuffer,
-  tile: I3STileOptions,
+  tileOptions: I3STileOptions,
```

Comment:
Should we group all the options parameters at the end of the function signature.